en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,274 @@
+stylesheets. This is done in two passes (two similar passes for both js and css).
+
+The first pass groups files by functionnalities. For instance all the javascript files related to theme
+are put in the same javascript file.
+The second pass groups all these 'functionnality' files in a unique file. The Back Office uses that
+final javascript (or stylesheet) file.
+
+In the case of our ``open-orchestra-media-admin-bundle``, the file concat.mediajs.js describes the
+first pass of the js concatenation: it glues all media javascripts together. The file concat.mediacss.js
+do the same for stylesheets. So the things you have to do is to tell grunt 
+when to lauch these tasks and how to add the two 'functionnality' files to the final javascript and
+stylesheet files.
+
+To add the media javascript task, modify the main javascript task (``grunt_task/javascript_task.js``):
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+        grunt.registerTask('javascript',
+            [
+                'coffee:discovering',
+                'coffee',
+                'concat:smartadminjs',
+                'concat:libjs',
+                'concat:orchestrajs',
+                'concat:mediajs',
+                'concat:js'
+            ]
+        );
+    };
+
+When the 'javascript' task will be run, the ``concat:mediajs`` task will be run.
+
+You can do the same for the media stylesheet by modifying the main css task (``grunt_task/css_task.js``):
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+        grunt.registerTask('css',
+            [
+                'less:discovering',
+                'less',
+                'concat:smartadmincss',
+                'concat:libcss',
+                'concat:orchestracss',
+                'concat:mediacss',
+                'concat:css',
+                'cssmin'
+            ]
+        );
+    };
+
+To include the result of the first pass to the final javascript file, alter the
+``grunt_tasks/options/concat.js.js`` file:
+
+.. code-block:: javascript
+
+    module.exports = {
+        src: [
+            'web/built/smartadmin.js',
+            'web/built/lib.js',
+            'web/built/orchestra.js',
+            'web/built/media.js'
+        ],
+        dest: 'web/js/all.js'
+    };
+
+That way, the final javascript file (``all.js``) will include the smartadmin functionnality, followed
+by the lib functionnality, and so on...
+
+You can do the same thing with stylesheets by modifying the ``grunt_tasks/options/concat.css.js``
+file:
+
+.. code-block:: javascript
+
+    module.exports = {
+        src: [
+            'web/built/smartadmin.css',
+            'web/built/lib.css',
+            'web/built/orchestra.css',
+            'web/built/media.css'
+        ],
+        dest: 'web/css/all.css'
+    };
+
+At that point, all is connected, you can run the grunt command (``./node_modules/.bin/grunt``) to
```

Comment:
Run the ``Grunt`` command () to regenerate the ``all.js`` and ``all.css`` filles. You could check that those files contains the assets of the ``media-admin-bundle``